sdk/src/client/api/block_builder/input_selection/mod.rs

```diff
@@ -228,6 +230,12 @@ impl InputSelection {
         self
     }
 
+    /// Sets the total mana rewards.
```

Comment:
Does this make sense then?
```suggestion
    /// Sets the total mana rewards for required inputs.
```